src/lib/config.ts

```diff
@@ -934,16 +934,11 @@ export class default_config {
      * Set this to something weird if you want to have fun every time Tridactyl tries to update its native messenger.
      *
      * %TAG will be replaced with your version of Tridactyl for stable builds, or "master" for beta builds
-     */
-    nativeinstallcmd =
-        "curl -fsSl https://raw.githubusercontent.com/tridactyl/tridactyl/master/native/install.sh -o /tmp/trinativeinstall.sh && sh /tmp/trinativeinstall.sh %TAG"
-
-    /**
-     * Set this to something weird if you want to have fun every time Tridactyl tries to update its native messenger.
      *
-     * Replaces %WINTAG with "-Tag $TRI_VERSION", similarly to [[nativeinstallcmd]].
+     * NB: Windows has its own platform-specific default - this is used only for Linux and OSX
      */
-    win_nativeinstallcmd = `powershell -NoProfile -Command "(New-Object System.Net.WebClient).DownloadFile('https://raw.githubusercontent.com/tridactyl/tridactyl/master/native/win_install.ps1', """$env:temp/tridactyl_installnative.ps1""")" & powershell -NoProfile -File %temp%\\tridactyl_installnative.ps1 %WINTAG & del %temp%\\tridactyl_installnative.ps1`
+    nativeinstallcmd =
```

Comment:
We can remove "blocked by testing on OSX" but just sticking an OSX default here that uses the old native messenger.

Probably worth doing that because I think there's nothing holding Linux back now.